cala-ledger-core-types/src/primitives.rs

```diff
@@ -156,6 +156,9 @@ impl Currency {
             Currency::Crypto(c) => c.code,
         }
     }
+
```

Comment:
Can you put the const at the top of the impl block

Reply:
Done.